test/e2e/framework/exec_util.go

```diff
@@ -33,6 +33,14 @@ import (
 // ExecCommandInContainer execute a command in the specified container.
 // Pass in stdin, tty if needed in the future.
 func (f *Framework) ExecCommandInContainer(podName, containerName string, cmd ...string) string {
+	stdout, stderr, err := f.ExecCommandInContainerWithFullOutput(podName, containerName, cmd...)
+	Logf("Exec stderr: %q", stderr)
+	Expect(err).NotTo(HaveOccurred(), "post request failed")
```

Comment:
That is the behavior before this PR though. 
I think the original author assumes that the err could only be related to the POST call. But it is not true though.